src/components/MediaHorizontalList.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <div>
+        <b-container class="container">
+            <div v-for="mediaObj in mediaObjs" :key='mediaObj.ids.imdb'>
+                <MediaTile class='mx-2'
+                           :title="mediaObj.title"
+                           :year="mediaObj.year"
+                           :img-source-uri="mediaObj.thumb"/>
+            </div>
+        </b-container>
+    </div>
+</template>
+
+<script lang="ts">
+    import {Component, Prop, Vue} from "vue-property-decorator";
+    import MediaTile from "@/components/MediaTile.vue";
+    import Movie from "@/models/Movie";
+
+    @Component({
+        components: {
+            MediaTile,
+        },
+    })
+    export default class MediaHorizontalList extends Vue {
+        @Prop() private mediaObjs: Movie[] = [];
+    }
+</script>
+
+<style scoped>
+    .container {
+        display: flex;
+        overflow-x: auto;
+        min-width: 100%;
+    }
+    ::-webkit-scrollbar {
```

Comment:
This works for Chome and Safari (webkit-based), for MS Edge and Firefox do this:
Edge: `-ms-overflow-style: -ms-autohiding-scrollbar;`
Firefox: `scrollbar-width: none;`
on .container

Reply:
Nice catch -- I only tested with Chrome and Safari. 

However with this code:
```suggestion
    .container {
        display: flex;
        overflow-x: auto;
        min-width: 100%;
        -ms-overflow-style: -ms-autohiding-scrollbar; /* Edge */
        scrollbar-width: none; /* Firefox */
    }

    ::-webkit-scrollbar {
        display: none;
    }
```

WebStorm complains: 
>Unknown CSS property 'scollbar-width'

Did you test it with Firefox?

Reply:
On further inspection, `scrollbar-width` "is part of the CSS Working Group's Scrollbars Module Level 1 draft, which is still a work in progress". It has experimental support in Firefox, but no other browsers:
https://caniuse.com/#search=scrollbar-width

Hence the complaint.